net/net-snmp/files/snmpd.init

```diff
@@ -210,6 +210,43 @@ snmpd_engineid_add() {
 	[ -n "$engineidnic" ] && echo "engineIDNic $engineidnic" >> $CONFIGFILE
 }
 
+snmpd_sink_add() {
+	local cfg="$1"
+	local section="$2"
+	local community
+	local port
+	local host
+
+	config_get host "$cfg" host
+	[ -n "section" -a -n "$host" ] || return 0
```

Comment:
Not really clear to me what you mean with "should that say -n "$section" instead?"